bleak/backends/winrt/client.py

```diff
@@ -406,10 +408,10 @@ async def unpair(self) -> bool:
             Boolean on whether the unparing was successful.
 
         """
-
-        if self._requester.device_information.pairing.is_paired:
+        device_information=await DeviceInformation.create_from_id_async(self._requester.device_information.id)
```

Comment:
Can we add a comment here explaining that the old `device_information` is out of date?

Reply:
Done